src/Features/Core/Portable/InlineHints/AbstractInlineParameterNameHintsService.cs

```diff
@@ -253,5 +258,11 @@ static bool SuffixMatchesParameterName(ReadOnlyMemory<char> suffix, string param
                        suffix.Span[1..].Equals(parameterName.AsSpan()[1..], StringComparison.Ordinal);
             }
         }
+
+        private bool ParameterMatchesArgumentName(SyntaxNode argument, IParameterSymbol? parameter, ISyntaxFactsService syntaxFacts)
+        {
+            var argumentName = GetArgumentName(argument, syntaxFacts);
+            return argumentName.Equals(parameter?.Name);
```

Comment:
Should this be case sensitive? Should that be an option?

Reply:
for VB it likely should be case sensitive.  For C# it's deabteable, but probably not.